oddt/utils.py

```diff
@@ -18,3 +19,48 @@ def is_rdkit_molecule(obj):
     """Check whether an object is an `oddt.toolkits.rdk.Molecule` instance."""
     return (hasattr(oddt.toolkits, 'rdk') and
             isinstance(obj, oddt.toolkits.rdk.Molecule))
+
+
+def check_molecule(mol,
+                   force_protein=False,
+                   force_coords=False,
+                   non_zero_atoms=False,
+                   *args):
+    """Universal validator of molecule objects. Usage of positional arguments is
+    allowed only for molecule object, otherwise it is prohibitted (i.e. the
+    order of arguments **will** change). Desired properties of molecule are
+    validated based on specified arguments. By default only the object type is
+    checked. In case of discrepancy to the specification a `ValueError` is
+    raised with appropriate message.
+
+    Parameters
+    ----------
+        mol: oddt.toolkit.Molecule object
+            Object to verify
+
+        protein: oddt.toolkit.Molecule object or None
+            Protein object to be used. If None, then the default one
+            is used, else the protein is new default.
+
+    """
+    if len(args) > 0:
```

Comment:
It won't work, `check_molecule(mol, True)` won't raise an error. I don't think it's possible to achieve expected behavior in py2 without using `**kwargs`.
Anyway, I think you should remove `*args`, it doesn't help us and it might be confusing for the user.